addons/crm/views/res_partner_views.xml

```diff
@@ -47,4 +47,17 @@
             </field>
         </record>
 
+        <record id="view_res_partner_crm_simple_form" model="ir.ui.view">
```

Comment:
Note that we could standardize new the record ID, even if it does not match the origin xml ID. Something like ``res_partner_view_form_simple_form``, and ``res.partner.view.form.simple.form.crm`` as name for example.

Reply:
Updated in CRM